android/MobiledgeXSDKDemo/app/release/output.json

```diff
@@ -1 +1 @@
-[{"outputType":{"type":"APK"},"apkData":{"type":"MAIN","splits":[],"versionCode":53,"versionName":"1.1.4","enabled":true,"outputFile":"MobiledgeXSDKDemo-release.apk","fullName":"release","baseName":"release"},"path":"MobiledgeXSDKDemo-release.apk","properties":{}}]
\ No newline at end of file
+[{"outputType":{"type":"APK"},"apkData":{"type":"MAIN","splits":[],"versionCode":54,"versionName":"1.1.5","enabled":true,"outputFile":"MobiledgeXSDKDemo-release.apk","fullName":"release","baseName":"release"},"path":"MobiledgeXSDKDemo-release.apk","properties":{}}]
```

Comment:
This file just notes values from the last time the Release version was built, so don't worry that it doesn't match what's in build.gradle.